consensus/mempool_test.go

```diff
@@ -246,17 +243,24 @@ func (app *CounterApplication) CreateBlock(
 		txValue := txAsUint64(tx)
 		if txValue != uint64(count) {
 			ret.InvalidTxs = append(ret.InvalidTxs, tx)
+			txResp := abcix.ResponseDeliverTx{
+				Code: code.CodeTypeBadNonce,
+				Log:  fmt.Sprintf("Invalid nonce. Expected %v, got %v", app.txCount, txValue),
+			}
+			ret.DeliverTxs = append(ret.DeliverTxs, &txResp)
```

Comment:
no we don't want our deliver tx to have invalid tx result, right? they can just be discarded as `ret.InvalidTxs`